lib/mixlib/install/options.rb

```diff
@@ -110,6 +110,10 @@ def latest_version?
         product_version.to_sym == :latest
       end
 
+      def partial_version?
+        !Mixlib::Versioning.parse(product_version) && !latest_version?
```

Comment:
Yes, but this method only gets executed once or twice when calling `#artifact_info`